core/libtexpdf-output.lua

```diff
@@ -117,30 +128,43 @@ SILE.outputters.libtexpdf = {
     -- position (cursorX + width - remember that the box's "width"
     -- is actually the shaped x_advance).
     if value.complex then
-      for i = 1, #(value.items) do
-        local glyph = value.items[i].gid
-        local buf = string.char(math.floor(glyph % 2^32 / 2^8)) .. string.char(glyph % 0x100)
-        pdf.setstring(cursorX + (value.items[i].x_offset or 0), cursorY + (value.items[i].y_offset or 0), buf, string.len(buf), font, value.items[i].glyphAdvance)
-        cursorX = cursorX + value.items[i].width
+      local x, y = self:getCursor()
+      y = SILE.documentState.paperSize[2] - y
+      for i = 1, #value.items do
+        local buf = glyph2string(value.items[i].gid)
+        self:setCursor(x + (value.items[i].x_offset or 0), y + (value.items[i].y_offset or 0))
```

Comment:
@simoncozens Off the top of your head can you tell me anything about what this line was supposed to do? I ported it across with the math as-is, but I'm concerned because I don't get what it does and deleting the line entirely doesn't break any of our test fixtures.